cmd/autoscaler/main.go

```diff
@@ -52,6 +52,8 @@ import (
 	asmetrics "knative.dev/serving/pkg/autoscaler/metrics"
 	"knative.dev/serving/pkg/autoscaler/scaling"
 	"knative.dev/serving/pkg/autoscaler/statserver"
+	v1 "knative.dev/serving/pkg/client/informers/externalversions/serving/v1"
+	revisioninformer "knative.dev/serving/pkg/client/injection/informers/serving/v1/revision"
```

Comment:
Why does the autoscaler have a revision informer 🤔

Reply:
We need to get the revision initial scale in the `Scale(...)` function

Reply:
We should probably propagate the annotation into the PA (like we do for minscale and friends too). Would that solve that issue?

Reply:
Currently we do propagate the initial scale annotation into PA, but in order to solve this problem, we will need to propagate it into `DeciderSpec` too because `scaling.Autoscaler` currently don't have access to the PA.

Reply:
It's fine. We can update Decider spec.